malboxes/profile-example.js

```diff
@@ -2,6 +2,13 @@
     "package": [{"package": "thunderbird"}],
     "document": [{"modtype": "add", "docpath": "C:\\Test.doc"}],
     "directory": [{"modtype": "add", "dirpath": "C:\\mlbxs\\"}],
+    "shortcut": [
+        //Create shortcuts for exe files
+        {"dest": "$env:USERPROFILE\\Desktop\\Fiddler.lnk", "target": "$env:LOCALAPPDATA\\Programs\\Fiddler\\Fiddler.exe"},
+        {"dest": "$env:USERPROFILE\\Desktop\\Fiddler Port 9999.lnk", "target": "$env:LOCALAPPDATA\\Programs\\Fiddler\\Fiddler.exe", "arguments": "/port:9999"},
```

Comment:
One of these exemples should be a desktop shortcut with the shorter syntax since you explicitly support it.